src/hb-coretext.cc

```diff
@@ -1244,8 +1244,6 @@ _hb_coretext_shape (hb_shape_plan_t    *shape_plan,
 	pos->x_offset = info->var1.i32;
 	pos->y_offset = info->var2.i32;
 
-	info->mask = HB_GLYPH_FLAG_UNSAFE_TO_BREAK;
```

Comment:
Hah! I didn't even remember that I have had added these lines to these shapers. Thanks for cleaning.